samples/react/ElmishCalc/src/State.fs

```diff
@@ -0,0 +1,77 @@
+module Calc.State
+
+open Calc.Types
```

Comment:
When opening modules at the "same level", you could simply do `open Types`.
This allow an easier read I think.

And only specify the full namespace when using nested component :)

This comment apply to all `open Calc.xx`

This type of writing is mostly a personal preference I think.